C_CPP/Pattern/README.md

```diff
@@ -0,0 +1,265 @@
+ }
+}
+ 
+ ```
+ --------------------------------------------------------------------------------------
+
+ * ### Pattern_5:
+ ![num5](https://user-images.githubusercontent.com/87390353/135322532-d08b19db-f807-4d91-bbd3-38a6ced676b8.jpg)
+ 
+ ### 🖊Code:
+ ```
+ #include<stdio.h>
+void main()
+{int i,k,j;
+for(i=1;i<=4;i++)
+  {k=1;
+for(j=1;j<=7;j++)
+ {
+if(j>=5-i&&j<=3+i)
+    {
+        printf("%d",k);
+        j<4?k++:k--;
+    }
+    else
+        printf(" ");
+ }
+        printf("\n");
+  }
+}
+ 
+ ```
+ --------------------------------------------------------------------------------------
+ ========================================================================================
+ 
+## 📌STAR PATTERNS📄
+* ### Pattern_1:
+![pattern_5](https://user-images.githubusercontent.com/87390353/135323734-c5f9b102-a837-4181-a9ee-6414475cfdbf.jpg)
+
+### 🖊Code:
+```
+#include<stdio.h>
+void main()
+{int j,i;
+  for(i=1;i<=5;i++)
+ {for(j=1;j<=4;j++)
+  {if (j>=6-i&&j<=10-i)
+    printf("*");
+    else
+        printf(" ");
+  }
+    printf("\n");
+ }
+}
+
+```
+--------------------------------------------------------------------------------------
+* ### Pattern_2:
+![pattern_4](https://user-images.githubusercontent.com/87390353/135324071-67c4495f-d85b-443c-bbb4-9c64fc86bfb4.jpg)
+
+### 🖊Code:
+```
+#include<stdio.h>
+void main()
+{
+  int i,j;
+  for(i=1;i<=6;i++)
+    {
+     for(j=1;j<=11;j++)
+        if(j>=7-i&&j<=5+i)
+
+
+        printf("*");
+
+
+        else
+        printf(" ");
+        printf("\n");
+
+}
+}
+```
+--------------------------------------------------------------------------------------
+* ### Pattern_3:
+![pattern_3](https://user-images.githubusercontent.com/87390353/135325654-645df4a3-4b7f-4100-920a-fd74f7b30a6a.jpg)
+
+### 🖊Code:
+```
+#include<stdio.h>
+void main()
+{int i,j,k=0;
+  for(i=1;i<=7;i++)
+ {
+     (i<=4?k++:k--);
+  for(j=1;j<=7;j++)
+  {if(j>=5-k&&j<=3+k)
+    printf("*");
+  else
+    printf(" ");
+  }
+    printf("\n");
+ }
+}
+
+```
+--------------------------------------------------------------------------------------
+* ### Pattern_4:
+![pattern_2](https://user-images.githubusercontent.com/87390353/135326627-5d91cd14-1719-4d20-a148-d71e2b3522c5.jpg)
+
+## 🖊Code:
+```
+#include<stdio.h>
+void main()
+{
+int i,j;
+for(i=1;i<=5;i++)
+ {
+ for(j=1;j<=9;j++)
+ 
+   {  if(j<=6-i||j>=4+i)
+   printf("*");
+else
+   printf(" ");
+   }
+   printf("\n"); 
+ }
+}
+
+```
+--------------------------------------------------------------------------------------
+* ### Pattern_5:
+![pattern_1](https://user-images.githubusercontent.com/87390353/135326959-c05ef3d9-7772-427d-a76d-21c686b628ba.jpg)
+
+### 🖊Code:
+```
+#include<stdio.h>
+void main()
+{
+int i,j;
+for(i=1;i<=5;i++)
+ {
+ for(j=1;j<=9;j++)
+   {  if(j<=i||j>=10-i)
+   printf("*");
+else
+   printf(" ");
+   }
+   printf("\n"); 
+ }
+}
+```
+### Hope these problems will help you✨.
+## Thank You😁
+
+  -Created by *[Anushka Paul](https://github.com/pilipi-puu-puu)*

```

Comment:
anushka, mentioning your name or id is not allowed. Kindly remove this.
rest of things are good, you can mention some comments as well for easy understanding.

Reply:
@deeqakkk ok